apps/flipper_auth/pubspec.yaml

```
@@ -0,0 +1,79 @@
+  equatable: ^2.0.3
+  supabase_flutter: ^2.9.1
+  otp: ^3.1.4
+  mobile_scanner: ^7.0.0
+  app_links: ^6.3.2
+  path_provider: ^2.1.4
+  shared_preferences: ^2.3.2
+  url_launcher: ^6.3.1
+  json_annotation: ^4.9.0
+  flipper_models: any
+dependency_overrides:
+  image: any
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 1 but found 2

(indentation)